pkg/setting/setting_plugins_test.go

```diff
@@ -271,4 +271,176 @@ func Test_readPluginSettings(t *testing.T) {
 			})
 		}
 	})
+
+	t.Run("when GF_INSTALL_PLUGINS is set", func(t *testing.T) {
```

Comment:
I think these tests are a leftover or something? You are not testing anything related to `GF_INSTALL_PLUGINS` as far as I can see